app/config/services.yml

```diff
@@ -1,18 +1,13 @@
 services:
-    # First we define some basic services to make these utilities available in
-    # the entire application
-    slugger:
```

Comment:
In a real application, these services would be used in other services, in console commands, etc.  You cannot remove them.